modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -0,0 +1,92 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distereoibution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_STEREO_HPP
+#define OPENCV_GAPI_STEREO_HPP
+
+#include <opencv2/calib3d.hpp>
```

Comment:
Strong dependency. :(

Reply:
Also, is it really required in this header?

Reply:
Got you 

Its needed for `cv::Ptr<cv::StereoBM>` from `StereoSetup`